MTTabController.m

```diff
@@ -40,4 +40,10 @@ - (BOOL) MouseTerm_resignFirstResponder
     return YES;
 }
 
+- (void) MouseTerm_tabControllerDealloc
+{
+    [[(TTTabController *)self profile] release];
```

Comment:
@cHoco In my environment, this causes crash if [osc11_set() function](https://github.com/saitoha/mouseterm-plus/blob/master/MTParser.m#L236) have been called already.
It can be reproduced by the following command in a single tab window.

```
$ echo -en '\033]11;darkgreen\007'
$ exit   # or close window
```

Can you also reproduce it?